components/uniset/src/enum_props.rs

```diff
@@ -0,0 +1,673 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+
+#[derive(Clone, PartialEq, Debug)]
+pub enum EnumeratedProperty {
```

Comment:
Are there well-defined discriminant values for these properties?  Take the ones from ICU4C?

Reply:
Filed followup issue #537 for this.